build/charts/antrea/values.yaml

```diff
@@ -388,6 +388,97 @@ multicluster:
     # -- WireGuard tunnel port for cross-cluster traffic.
     port: 51821
 
+nodeNetworkPolicy:
+  # Configure nodeNetworkPolicy privileged ingress rules for Antrea Agent. They could be applied to both IPv4 and IPv6.
+  privilegedIngressRules:
+    - protocol: tcp
+      ports:
+      - 22
+      comment: "allow ingress traffic to SSH"
+    - protocol: tcp
+      ports:
+      - 6443
+      - 10248:10250
+      - 10256:10258
+      comment:  "allow ingress traffic to Kubernetes ports"
+    - protocol: tcp
+      ports:
+      - 10348:10351
+      comment:  "allow ingress traffic to Antrea ports"
+    - protocol: udp
+      ports:
+      - 51820:51821
+      comment: "allow ingress traffic to Wireguard ports"
+    - protocol: icmp
+      icmpType: 0
+      comment: "allow ICMP echo-reply traffic"
+    - protocol: icmp
+      icmpType: 8
+      comment: "allow ICMP echo-request traffic"
```

Comment:
It should only include **necessary ports to re-apply NodeNetworkPolicy** when something wrong is configured. It's not supposed to list all possible traffic, which is the NodeNetworkPolicy supposed to do.

Reply:
Is the **necessary ports to re-apply NodeNetworkPolicy** like SSH 22, Kubernetes api server port 6443?